chronos/asyncloop.nim

```diff
@@ -294,6 +294,9 @@ when defined(windows):
       cb*: CallbackFunc
       errCode*: OSErrorCode
       bytesCount*: uint32
+      cell*: ForeignCell # we need this `cell` to protect our `cb` environment,
```

Comment:
so, from what I can tell this should not be needed after all, ie a simple `GC_ref` should be sufficient:

although the callback that is registered with `RegisterWaitForSingleObject` might be called from a different thread, the actual callback that we implement does not perform any memory allocations or other interactions with the garbage collector - all it does is re-post the same notification to the thread that ultimately owns the cb that this cell aims to protect.

Reply:
Fixed.